web/yaamp/modules/site/common_results.php

```diff
@@ -127,8 +129,16 @@ function cmp($a, $b)
 	$ts = $isup ? datetoa2($stratum->started) : '';
 
 	echo '<tr class="ssrow">';
-	echo '<td style="background-color: '.$algo_color.'"><b>';
-	echo CHtml::link($algo, '/site/gomining?algo='.$algo);
+	
+	$stratum_instances = dboscalar("SELECT COUNT(*) FROM stratums WHERE algo='$algo'");
+        $stratum_urls = dbolist("SELECT url FROM stratums WHERE algo='$algo'"); // AND NOT(url = NULL)");
+
+        $urls = $stratum_instances." Stratums TCP Server:";
+        foreach ($stratum_urls as $stra)        {
+                $urls = $urls . "\n" . $stra['url'];
+        }
+        echo '<td style="background-color: '.$algo_color.'"><b title="'.$urls.'">';
+	echo CHtml::link($algo, '/site/Gostratums?algo='.$algo);
```

Comment:
This cap should be removed here.. like "gomining"... the actionGostratum cap is only a way to make action functions more readable.